src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/callback/CreateCaseValidator.java

```diff
@@ -84,7 +81,7 @@ public CreateCaseValidator() {
         Validation<String, List<ScannedDocument>> hasScannedDocuments = getScannedDocuments(caseDetails);
         Validation<String, List<OcrDataField>> hasOcrDataFields = getOcrDataFields(caseDetails);
 
-        List<Validation<String, ?>> validations = Arrays.asList(
+        Array<Validation<String, ?>> validations = Array.of(
```

Comment:
I think it's `Seq` interface ;) Same for errors var and around `getValidationErrors`

Reply:
Oh.. I'll change it